src/hooks/screen/useCreateChat.ts

```diff
@@ -57,7 +57,7 @@ const useCreateChat = () => {
     return chatData;
   };
 
-  const createSignChat = async (members: string[], selectedUser, from) => {
+  const createSignChat = async (members: string[], selectedUser, from: AllowedGoToChatScreen) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature for `createSignChat` has been updated to include a new parameter `from` of type `AllowedGoToChatScreen`. This change is good as it improves the type safety of the function. However, please ensure that all calls to this function are updated to pass this new argument.
```diff
- const createSignChat = async (members: string[], selectedUser, from) => {
+ const createSignChat = async (members: string[], selectedUser, from: AllowedGoToChatScreen) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->